themes/OneNord-color-theme.json

```diff
@@ -219,7 +219,7 @@
       "scrollbarSlider.background": "#434c5e99",
       "scrollbarSlider.hoverBackground": "#434c5eaa",
       "selection.background": "#81a1c199",
-      "sideBar.background": "#2A303B",
+      "sideBar.background": "#2e3440",
```

Comment:
Make the sidebar the same colour of the editor background. I think it's more pleasant to look at, WDYT?